opentelekomcloud/acceptance/ecs/resource_opentelekomcloud_compute_instance_v2_test.go

```diff
@@ -193,72 +113,17 @@ func TestAccComputeV2Instance_bootFromVolumeVolume(t *testing.T) {
 			{
 				Config: testAccComputeV2Instance_bootFromVolumeVolume,
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckComputeV2InstanceExists("opentelekomcloud_compute_instance_v2.instance_1", &instance),
+					testAccCheckComputeV2InstanceExists(resourceName, &instance),
 					testAccCheckComputeV2InstanceBootVolumeAttachment(&instance),
 				),
 			},
 		},
 	})
 }
 
-func TestAccComputeV2Instance_bootFromVolumeForceNew(t *testing.T) {
-	var instance servers.Server
-	var newInstance servers.Server
-
-	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { common.TestAccPreCheck(t) },
-		Providers:    common.TestAccProviders,
-		CheckDestroy: TestAccCheckComputeV2InstanceDestroy,
-		Steps: []resource.TestStep{
-			{
-				Config: testAccComputeV2Instance_bootFromVolume,
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckComputeV2InstanceExists(
-						"opentelekomcloud_compute_instance_v2.instance_1", &instance),
-				),
-			},
-			{
-				Config: testAccComputeV2Instance_bootFromVolumeUpdate,
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckComputeV2InstanceExists(
-						"opentelekomcloud_compute_instance_v2.instance_1", &newInstance),
-					testAccCheckComputeV2InstanceInstanceIDsDoNotMatch(&instance, &newInstance),
-				),
-			},
-		},
-	})
-}
-
-func TestAccComputeV2Instance_changeFixedIP(t *testing.T) {
-	var instance servers.Server
-	var newInstance servers.Server
-
-	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { common.TestAccPreCheck(t) },
-		Providers:    common.TestAccProviders,
-		CheckDestroy: TestAccCheckComputeV2InstanceDestroy,
-		Steps: []resource.TestStep{
-			{
-				Config: testAccComputeV2Instance_fixedIP,
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckComputeV2InstanceExists(
-						"opentelekomcloud_compute_instance_v2.instance_1", &instance),
-				),
-			},
-			{
-				Config: testAccComputeV2Instance_fixedIPUpdate,
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckComputeV2InstanceExists(
-						"opentelekomcloud_compute_instance_v2.instance_1", &newInstance),
-					testAccCheckComputeV2InstanceInstanceIDsDoNotMatch(&instance, &newInstance),
-				),
-			},
-		},
-	})
-}
-
```

Comment:
Why those tests were deleted?

Reply:
These tests check if instance recreated. They change arguments with `ForceNew: true` set

Reply:
In that case it makes sense to remove `update` step only, not the whole test.

Reply:
Or even just remove `testAccCheckComputeV2InstanceInstanceIDsDoNotMatch`